.github/workflows/ci.yml

```diff
@@ -12,8 +12,8 @@ defaults:
     shell: bash -l {0}
 
 env:
-  flavour: Oss
-  config: Debug
+  flavour: oss
+  config: debug
```

Comment:
It's used for Gradle tasks names, which are not case sensitive. However, we now use it for files as well, and the file names are in the lowercase.